src/mscorlib/model.xml

```diff
@@ -3434,6 +3434,7 @@
       <Member Status="ImplRoot" Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
       <Member Name="Add(System.IntPtr,System.Int32)" />
       <Member Name="Equals(System.Object)" />
+      <Member Status="ImplRoot" Name="System.IEquatable&lt;System.IntPtr&gt;.Equals(System.IntPtr)" />
```

Comment:
@jkotas You appear to be right; I've removed the entries entirely from model.xml.